smoke-tests/build.gradle.kts

```
@@ -81,6 +79,8 @@ tasks {
 
     val shadowTask = project(":javaagent").tasks.named<ShadowJar>("shadowJar").get()
     inputs.files(layout.files(shadowTask))
+      .withPropertyName("javaAgent")
```

conventionally in this repo we have been using javaagent as one word
```suggestion
      .withPropertyName("javaagent")
```